src/pages/gd/Stake/StakeDeposit/index.tsx

```diff
@@ -293,11 +299,18 @@ const StakeDeposit = ({ stake, onDeposit, onClose, activeTableName }: StakeDepos
                     <div className="text-center mt-4">
                         {activeTableName === 'GoodDAO Staking' ? (
                             i18n._(
-                                t`You have just staked your G$s towards our GoodDAO, this action is gonna reward you with GOOD governance tokens, which are non-transferable so can't be traded.`
+                                t`You have just staked your G$s towards our GoodDAO, 
+                                this action is gonna reward you with GOOD governance tokens, 
+                                which are non-transferable so can't be traded.`
                             )
                         ) : (
                             <>
-                                {i18n._(t`Did you just create UBI for thousands of people around the world?`)}{' '}
+                                {state.loading ?
+                                i18n._(t`Your staking transaction will generate UBI for thousands of people around 
+                                         the world has just been broadcasted to the network. `)
+                              : i18n._(t`You are creating UBI to thousands of people around the world. `)
```

Comment:
no need for distinction

Reply:
it was discussed on the issue comment section what the text should be

these different types of text just indicate the process of pending/confirmed transactions
(if a user decides to wait with the dialog open)
now it also shows the pending indicator in the dialog itself

so all texts now update according to the status of the transaction
and this is tested, and works

But do I now interpret that you think we shouldn't indicate the process of transaction through the written texts?